.config/mkdocs.yml

```diff
@@ -19,10 +19,10 @@ nav:
       - about-civicactions/annual-summit.md
   - Employee benefits:
       - employee-benefits/README.md
-      - Personal Time Off (PTO): employee-benefits/README.md
+      - Personal Time Off (US employees): employee-benefits/README.md
       - employee-benefits/us-compensation.md
       - employee-benefits/us-tech-stipend.md
-      - employee-benefits/canada-benefits-policy.md
+      - Personal Time Off (Canadian employees): employee-benefits/pto-canada.md
```

Comment:
@ContentBrewmaster I am not clear why we need the titles here, if they are the same as the page headers?

Reply:
@grugnog you reminded me that the new title on each page is populating the nav.

Reply:
@grugnog can we get your approval please?